packages/components/src/List/ListComposition/SortBy/SortBy.component.js

```diff
@@ -40,7 +40,7 @@ function SortBy(props) {
 	const onOrderChange = event =>
 		performChange(event, { ...currentValue, isDescending: !currentValue.isDescending });
 
-	return (
+	return displayMode !== 'table' ? (
```

Comment:
We can return `null` before all of that stuff by using
```javascript
if (displayMode === DISPLAY_MODE.TABLE) {
  return null;
}

useEffect();  // bla bla bla
```

Reply:
done